### ISSUE TITLE
Uploading a file to remote does not work on Firefox & Selenium Grid

### ISSUE THREAD

**Issue by wskorodecki** [SeleniumHQ/selenium#7819]
## 💥 Regression Report

File upload does not work on Firefox when using Selenium Grid, `remote.FileDetector` and a file stored locally:
```
WebDriverError: HTTP method not allowed
at parseHttpResponse ([...]/node_modules/selenium-webdriver/lib/http.js:580:11)
    at Executor.execute ([...]/node_modules/selenium-webdriver/lib/http.js:489:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
``` 
This is occurs after sending `uploadFile` command.

But it works without any problems when switching to old Selenium Server Standalone. For me it looks like the FileDetector is not doing what it should or it does it incorrectly because when I use a path to the file stored on remote machine (`fileInput.sendKeys('C:\\test.txt')`) then it works on both servers.

## Last working

Worked up to: `selenium-server-standalone-3.141.59.jar`
Stopped working in: `selenium-server-4.0.0-alpha-3.jar`

## To Reproduce

- Start geckodriver on remote: `C:\geckodriver-v0.26.0-win64\geckodriver.exe --host=0.0.0.0 --port=4444 --log trace`
- Start Selenium Grid on remote: `java -jar selenium-server-4.0.0-alpha-3.jar standalone --host 0.0.0.0 --port 4445`
- Start Selenium Server Standalone on remote: `java -jar selenium-server-standalone-3.141.59.jar -host 0.0.0.0 -port 4446 -role standalone --debug`
- Create `test.html` file and make it accessible via test vhost `my-host.test`:
```
<!doctype html>
<html>
<head>
    <meta charset="utf-8">
</head>
<body>
    <input type="file" name="file" id="file" />
</body>
</html>
```
- Add `my-host.test` vhost with correct IP to `C:\Windows\System32\drivers\etc\hosts` file
- Create `test.txt` file with some contents (for example: `test`) and make it available relatively to the `test.js` file: `data/files-to-upload/test.txt`
- The same text file should be available on remote too: `C:\test.txt`

## Expected behavior

The input present on the test page should receive a path to the `test.txt` file.

## Test script reproducing this issue

```
const { Builder, By, Browser } = require('selenium-webdriver');
const remote = require('selenium-webdriver/remote');

(async () => {
  const builder = new Builder()
    .forBrowser(Browser.FIREFOX);

  // Connecting to Selenium Grid (4.*)
  builder.usingServer('http://192.168.0.132:4445');

  const driver = builder.build();
  driver.setFileDetector(new remote.FileDetector());

  await driver.get('http://my-host.test/test.html');

  //const filePath = 'C:\\test.txt'; // Uploading locally stored file works.
  // File transferred from local to remote does not work.
  const filePath = `${__dirname}/../data/files-to-upload/test.txt`;
  const fileInput = await driver.findElement(By.id('file'));
  await fileInput.sendKeys(filePath);
})();
```

## Test which confirms that the same works on old server

```
const { Builder, By, Browser } = require('selenium-webdriver');
const remote = require('selenium-webdriver/remote');

(async () => {
  const builder = new Builder()
    .forBrowser(Browser.FIREFOX);

  // Connecting to Selenium Server Standalone (3.*)
  builder.usingServer('http://192.168.0.132:4446/wd/hub');

  const driver = builder.build();
  driver.setFileDetector(new remote.FileDetector());

  await driver.get('http://my-host.test/test.html');

  // File transferred from local to remote.
  const filePath = `${__dirname}/../data/files-to-upload/test.txt`;
  const fileInput = await driver.findElement(By.id('file'));
  await fileInput.sendKeys(filePath);
})();
```

### Remote:
OS: Windows 10 Pro 64bit PL
Browser: Firefox 64bit
Browser version: 70.0.1
Browser Driver version: GeckoDriver 0.26
Language Bindings version: JavaScript 4.0.0-alpha.5 https://www.npmjs.com/package/selenium-webdriver/v/4.0.0-alpha.5
Java: OpenJDK Runtime Environment (build 13.0.1+9)
Selenium Server Standalone version: 3.141.59
Selenium Grid version: 4.0.0 alpha 3

**Comment by diemol** [comment]
I think @barancev already fixed this and it will be released in the next alpha. Please correct me if I'm wrong @barancev

**Comment by barancev** [comment]
Yes, the fix is waiting for the next release: b514a7490e298aecffc720a55f091b824a7658b4
